### DISCUSSION TITLE
QR Code Op with Text input

### DISCUSSION THREAD

**Discussion by dererdmann** [cables-gl/cables_docs#235] [! Welcome - readme first!]
The [QR Code Op](https://cables.gl/op/Ops.Gl.QrCode) has the input type number, which makes it impossible to input urls or alphanumeric IDs. Should this be a type String instead? Or even both?
<img width="350" alt="showing details of the op QR Code" src="https://user-images.githubusercontent.com/9743261/167365789-385269c4-24a3-4002-8b66-6a9a315bcd71.png">

**Comment by pandrr** [comment]
thanks, will be fixed in the next release

for now you can use this converter to set the url:  https://dev.cables.gl/op/Ops.String.StringOld2New
